src/api/network/blockchain/Imp.cpp

```diff
@@ -378,26 +367,22 @@ auto BlockchainImp::Init(
         "tcp://metier1.opentransactions.org:8814",
         "tcp://metier2.opentransactions.org:8814",
     };
-    const auto existing = [&] {
-        auto out = Set<CString>{};
 
-        for (const auto& server : GetSyncServers()) {
-            // TODO GetSyncServers should return pmr strings
-            out.emplace(server.c_str());
-        }
+    Set<CString> existing{};
 
-        for (const auto& server : defaultServers) {
-            if (0 == out.count(server)) {
-                if (false == api_.GetOptions().TestMode()) {
-                    AddSyncServer(server);
-                }
+    for (const auto& server : GetSyncServers()) {
+        // TODO GetSyncServers should return pmr strings
+        existing.emplace(server.c_str());
```

Comment:
whats the benefit of using emplace rather than insert with c_string ?

Reply:
what is benefit of using insert rather than emplace with c_string?:P

Reply:
btw. I will move it into constructor of "existing"